app/controllers/user_auth_controller.rb

```diff
@@ -32,4 +32,18 @@ def user_not_authorized(exception)
   def authenticated_user_root_path
     current_user.only_invited? ? root_path : users_rdvs_path
   end
+
+  def verify_user_identity
+    return unless current_user.only_invited?
+    return if cookies.encrypted[:"user_#{current_user.id}_verified"] == true
```

Comment:
Suggestion de nom:

```suggestion
    return if cookies.encrypted[:"user_name_initials_verified_#{current_user.id}"] == true
```

?